### ISSUE TITLE
Breadcrumb is not displayed

### ISSUE THREAD

**Issue by fthommen** [my17560/dokuwiki-template-readthedokus#12]
It looks as if there were no breadcrumb displayed on the page.  The navigation only shows the "homepage" and the current page, but not the intermediate ones.  E.g. it looks like this with the readthedokus template:
![image](https://user-images.githubusercontent.com/279232/152659362-566a0c4c-2872-4323-b65a-75de86fd174c.png)

but like this when using the standard dokuwiki template:
![image](https://user-images.githubusercontent.com/279232/152659376-cd366fdb-46de-4353-bba8-fedeb61186a8.png)

**Comment by my17560** [comment]
This is intended because the original "Read the Docs" looks that way. However, it sounds reasonable to add an option whether to show the original breadcrumb. I'll work on it.

**Comment by fthommen** [comment]
Yes, from that perspective it makes sense. I'm still thinking more in DW terms than in Readthedocs terms :-)

**Comment by my17560** [comment]
I add the "Use Dokuwiki breadcrumbs" option. Enable it to show default DokuWiki breadcrumbs.

**Comment by fthommen** [comment]
Works like a charm.  Thanks a lot
